app/assets/javascripts/initializePage.js

```diff
@@ -36,6 +36,27 @@ function callInitializers() {
   initializeDateHelpers();
 }
 
+// This is a helper function that mitigates a race condition happening in JS
+// when this intializer runs before `base.jsx` gets to load the
+// window.Forem / window.Forem.Runtime utility class we depend on for
+// podcast/video playback
+function initializeRuntimeDependantFeatures() {
```

Comment:
Would it be possible to add this to a pack file instead since we're trying to get rid of the legacy JavaScript or is the existing functions that we're using too inter tangled? Just thinking out loud here since we're in the process of moving these initializers.

Reply:
This is a great mention to the migration effort underway. I just did my best to migrate them into the webpack bundle in a way that makes sense and so that the error caused by the `window.Forem` artifact being `undefined` won't appear anymore 🙂